README.md

```diff
@@ -1,64 +1,54 @@
-<img src="https://ucbrise.github.io/opaque/opaque.svg" width="315" alt="Opaque">
+<img src="https://mc2-project.github.io/opaque/opaque.svg" width="315" alt="Opaque">
 
 **Secure Apache Spark SQL**
 
-[![Build Status](https://travis-ci.org/ucbrise/opaque.svg?branch=master)](https://travis-ci.org/ucbrise/opaque)
+[![Build Status](https://travis-ci.org/mc2-project/opaque.svg?branch=openenclave)](https://travis-ci.org/mc2-project/opaque)
 
-Opaque is a package for Apache Spark SQL that enables encryption for DataFrames using Intel SGX trusted hardware. The aim is to enable analytics on sensitive data in an untrusted cloud. Once the contents of a DataFrame are encrypted, subsequent operations will run within SGX enclaves.
+Opaque is a package for Apache Spark SQL that enables encryption for DataFrames using the OpenEnclave framework. The aim is to enable analytics on sensitive data in an untrusted cloud. Once the contents of a DataFrame are encrypted, subsequent operations will run within hardware enclaves (such as Intel SGX).
 
-This project is based on our NSDI 2017 paper [1]. The oblivious execution mode is not included in this release.
+This project is based on the following NSDI 2017 paper [1]. The oblivious execution mode is not included in this release.
 
 This is an alpha preview of Opaque, which means the software is still in development (not production-ready!). It currently has the following limitations:
 
 - Unlike the Spark cluster, the master must be run within a trusted environment (e.g., on the client).
 
 - Not all Spark SQL operations are supported. UDFs must be [implemented in C++](#user-defined-functions-udfs).
 
-- Computation integrity verification (section 4.2 of the NSDI paper) is not included.
+- Computation integrity verification (section 4.2 of the NSDI paper) is currently work in progress.
 
 [1] Wenting Zheng, Ankur Dave, Jethro Beekman, Raluca Ada Popa, Joseph Gonzalez, and Ion Stoica.
 [Opaque: An Oblivious and Encrypted Distributed Analytics Platform](https://people.eecs.berkeley.edu/~wzheng/opaque.pdf). NSDI 2017, March 2017.
 
 ## Installation
 
-After downloading the Opaque codebase, build and test it as follows. (Alternatively, we offer a [Docker image](docker/) that contains a prebuilt version of Opaque.)
```

Comment:
If we're not going to keep offering the Docker image, should we also remove `docker/`?

Reply:
Removed.